evaluation/comparing_prompt_with_and_without_context.py

```diff
@@ -35,19 +35,32 @@ def make_prompts_files():
 
```

Comment:
Line 25: `input_text = "```{{chunk}}```"`
I know elsewhere we've changed chunk objects from strings to dicts, so it may be worth checking we're actually just passing in the string here. Otherwise it may need to be something like `chunk['text']`.

Reply:
I think here, this is telling promptfoo where to put the "chunk" variable it's been given, which will be a string. But that's a very useful heads up for where we make the tests file, thanks!